l10n_br_account_payment_mode/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Brazil Account Banking - Payments Mode
+======================================
+
+This module provide an mode payment for payment orders.
```

Comment:
s/provide/provides
s/an mode for payment orders/a mode for payment orders in Brazil